i18n/create_json.js

```diff
@@ -12,6 +12,7 @@
  *            2020-03-30: [AK] updated to use xml2js module
  *            2020-03-31: [AK] updated code to use default value for missing in current language
  *            2020-04-01: [AK] updated default language to 'values-en-rAU'
+ *            2020-04-10: [GS] included and updated default language to 'values-en-rUS
```

Comment:
```suggestion
 *            2020-05-10: [GS] included and updated default language to 'values-en'
```

Reply:
Thanks, well spotted. Changed it to your suggestion.